File: src/main/scala-graphql/io/github/graphql/scala/resolver/CommitCommentResolver.scala
package io.github.graphql.scala.resolver

import io.github.graphql.scala.model._
import ReactionContentTO._

/**
 * Resolver for CommitComment
 */
@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-07T19:49:37+0800"
)
trait CommitCommentResolver {

    @javax.validation.constraints.NotNull
    @throws[Exception]
    def reactions(commitCommentTO: CommitCommentTO, after: String, before: String, content: ReactionContentTO, first: Option[Int], last: Option[Int], orderBy: ReactionOrderTO): ReactionConnectionTO

    @throws[Exception]
    def userContentEdits(commitCommentTO: CommitCommentTO, after: String, before: String, first: Option[Int], last: Option[Int]): UserContentEditConnectionTO

}

File: src/main/java-graphql/io/github/graphql/resolver/CommitCommentResolver.java
package io.github.graphql.resolver;

import io.github.graphql.model.*;

/**
 * Resolver for CommitComment
 */
@javax.annotation.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-03-06T19:47:48+0800"
)
public interface CommitCommentResolver {

    @javax.validation.constraints.NotNull
    ReactionConnectionTO reactions(CommitCommentTO commitCommentTO, String after, String before, ReactionContentTO content, Integer first, Integer last, ReactionOrderTO orderBy) throws Exception;

    UserContentEditConnectionTO userContentEdits(CommitCommentTO commitCommentTO, String after, String before, Integer first, Integer last) throws Exception;

}
